objectDataTypes.md

```diff
@@ -7,145 +7,182 @@
 
 Without putting the code into Ruby or IRB predict the output and data-type of the following?
 
+1.  Output __________   Answer: _________  
+
+  ```ruby
+  num = 27
+  num = num + 5
+  num
+  ```
+
+1.  Output __________   Answer: _________
+
+
+  ```ruby
+  num = 2
+  num = num + num + 2
+  num
+  ```
+1.  Output __________   Answer: _________
 
-```ruby
-num = 27
-num = num + 5
-num
-```
-
-```ruby
-num = 2
-num = num + num + 2
-num
-```
 	 
-```ruby
-num = "4"
-num = num + num
-num
-```
-
-```ruby
-stack_teacher = "Jamie"
-queue_teacher = "Kari"
-c7_teachers = stack_teacher + " " + queue_teacher
-c7_teachers
-```
-
-```ruby
-num = 2
-num = num * num
-num = num * num
-num
-```
-
-```ruby
-num = 27
-answer = num % 3
-answer
-```
-
-```ruby
-num = 100
-num = num / 2
-num = num / 2
-num = num / 2
-num
-```
-
-
-```ruby
-num = 100.0
-num = num / 2
-num = num / 2
-num = num / 2
-num
-```
-
-```ruby
-num = 5.0
-num = num * 2 + 3
-num = num / 2
-```
-
-
-```ruby
-answer = :dan
-answer
-```
-
-```ruby
-list = ["Pikachu", "Bulbasaur", "Squirtle", "Charizard"]
-list
-```
-
-```ruby
-list = ["Pikachu", "Bulbasaur", "Squirtle", "Charizard"]
-list[3]
-```
-
-```ruby
-list = ["Pikachu", "Bulbasaur", "Squirtle", "Charizard"]
-list[1]
-```
-```ruby
-list = ["Pikachu", "Bulbasaur", "Squirtle", "Charizard"]
-list.length
-```
-
-```ruby
-list = ["Pikachu", "Bulbasaur", "Squirtle", "Charizard"]
-list.first
-```
-
-```ruby
-list = ["Pikachu", "Bulbasaur", "Squirtle", "Charizard"]
-list[4]
-```
-
-```ruby
-list = ["Pikachu", "Bulbasaur", "Squirtle", "Charizard"]
-list[-1]
-```
-
-
-```ruby
-pikachu = {
-	name: "Pikachu",
-	type: :electric,
-	species: "Mouse Pokemon",
-	height: .41
-	weight: 6.0
-	id: 25
-}
-pikachu
-```
-
-
-```ruby
-pikachu = {
-	name: "Pikachu",
-	type: :electric,
-	species: "Mouse Pokemon",
-	height: .41
-	weight: 6.0
-	id: 25
-}
-pikachu[:name]
-```
-
-```ruby
-pikachu = {
-	name: "Pikachu",
-	type: :electric,
-	species: "Mouse Pokemon",
-	height: .41
-	weight: 6.0
-	id: 25
-}
-pikachu[:id]
-```
+  ```ruby
+  num = "4"
+  num = num + num
+  num
+  ```
+
+1.  Output __________   Answer: _________
+
+  ```ruby
+  stack_teacher = "Jamie"
+  queue_teacher = "Kari"
```

Comment:
We should switch these around as it turns out the class names are opposite of what we had thought.